lang/en/docs/tutorials/dft/chemical/molecule-surface.md

```diff
@@ -0,0 +1,89 @@
+
+We start with [opening](../../../entities-general/actions/create.md) an instance of the [Materials Designer Interface](../../../materials-designer/overview.md) for creating and designing new [Materials structures](../../../materials/overview.md) on our platform. 
+
+## Create a Gold Surface 
+
+In order to create a gold surface, the user should first [import](../../../materials-designer/header-menu/input-output/import.md) a sample crystalline structure of pure gold into the current Materials Designer session, from the account-owned [collection](../../../accounts/collections.md) of materials. 
+
+The instructions contained [in this page](../../../materials-designer/header-menu/advanced/surface-slab.md) should then be followed in order to create a surface of Gold, with normal vector oriented along the [211] axis, using our surface creator in Materials Designer, starting from the original gold crystalline sample. 
+
+## Create Benzene Molecule Entry in Materials Collection
+
+The chemical structure for Benzene can readily be [imported](../../../materials/actions/copy-bank.md) from the [Materials Bank](../../../materials/bank.md) into the account-owned [collection](../../../accounts/collections.md), if it is not already present there. 
+
+Alternatively, the above-mentioned POSCAR structure can be manually [uploaded](../../../materials/actions/upload.md) by the user into the materials collection, after saving its data contents into a new file on the local disk.
+ 
+## Import the Benzene Molecule into Materials Designer
+
+The Benzene molecule should now be [imported](../../../materials-designer/header-menu/input-output/import.md) into the current Materials Designer session, from the account-owned [collection](../../../accounts/collections.md) of materials. 
+
+Once imported into Materials Designer, the benzene molecule will appear as a new distinct entry item within the list of structures shown on the [left-had items list sidebar](../../../materials-designer/sidebar-items.md) of the Designer interface, besides the previously-generated gold surface.
+ 
+Care should be taken by the user to [remove](../../../materials-designer/sidebar-items.md#delete-item) any other material structure entry listed in the sidebar, besides benzene and the gold surface being considered here, that may have been loaded by [default](../../../materials/default.md) initially at the moment of the opening of Materials Designer.
+
+## Open Multi-Materials 3D Editor
+
+After both the benzene molecule and the gold surface have been created as two separate structural items into the current session of Materials Designer, the user should now open an instance of the [Multi-Materials 3D Editor](../../../materials-designer/3d-editor/edit.md) via the ["View" Menu](../../../materials-designer/header-menu/view.md#multi-material-3d-editor), located within the [header bar](../../../materials-designer/header-menu/header-menu-intro.md) of the Materials Deigner Interface.
```

Comment:
"into the current session of Materials Designer" => "in the current session of Materials Designer